sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -771,6 +778,16 @@ class UnsupportedOperationsSuite extends SparkFunSuite {
     }
   }
 
+  /** Assert that the logical plan is supportd for continuous procsssing mode */
```

Comment:
nit:`supportd` -> `supported`

Reply:
ah, my bad!. Thanks for pointing it out.